pwdtk/middlewares.py

```diff
@@ -43,12 +43,7 @@ def must_renew_password(self, request):
         if request.method == 'GET':
             return False
 
-        if django.VERSION < (1, 10):
-            is_authenticated = request.user and request.user.is_authenticated()
-        else:
-            is_authenticated = request.user and request.user.is_authenticated
-
```

Comment:
I understand, why the if statement disappeared, byut are you sure now, that request.user is alwasy not None?

Reply:
not sure but i think it will be a Anonymus users at least, never None

Reply:
any way to test this quickly (that you get an anonymous user object)?